arroyo/processing/strategies/streaming/factory.py

```diff
@@ -31,8 +37,12 @@ def should_drop(self, message: Message[TPayload]) -> bool:
 
 class ConsumerStrategyFactory(ProcessingStrategyFactory[TPayload]):
     """
-    Builds a three step consumer strategy consisting of filter, transform
-    and collect phases.
+    Builds a four step consumer strategy consisting of dead letter queue,
+    filter, transform, and collect phases.
+
+    The `dead_letter_queue_policy` defines what to do when an bad message
+    is encountered throughout the next processing step(s). A DLQ wraps the
+    entire strategy and handles a specific exception to handle bad messages.
```

Comment:
I feel like this a little confusing to me, would it be fair to say something like?: 

`A DLQ policy wraps the entire strategy and will catch InvalidMessage exceptions and handle them as the policy dictates.`